extensions/amp-render/validator-amp-render.protoascii

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-render
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-render"
+    version: "1.0"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-render>
+  html_format: AMP
+  tag_name: "AMP-RENDER"
+  requires_extension: "amp-render"
+  attrs: {
+    name: "binding"
+    value: "always"
+    value: "no"
+    value: "refresh"
+  }
+  attrs: { name: "credentials" }
+  attrs: { name: "xssi-prefix" }
+  attrs: { name: "key" }
+  attrs: {
+    name: "data-amp-bind-src"
+    mandatory_anyof: "['src','[src]','data-amp-bind-src']"
+  }
+  attrs: {
+    name: "[src]"
+    mandatory_anyof: "['src','[src]','data-amp-bind-src']"
+  }
+  attr_lists: "extended-amp-global"
+  # spec_url: "https://amp.dev/documentation/components/amp-render/"
```

Comment:
please uncomment if we're expecting this url to be real. is there a doc PR doing that?